tests/func/test_repro.py

```diff
@@ -736,9 +736,9 @@ def test(self):
         os.mkdir(dname)
         foo = os.path.join(dname, self.FOO)
         bar = os.path.join(dname, self.BAR)
-        code = os.path.join(dname, self.CODE)
+        code_dir = os.path.join(dname, self.CODE_DIR)
```

Comment:
No need to modify unrelated tests 🙂  Just create what you need for your tests and don't touch other tests.